Pipes - Kate Evans-Spitzer - Solar System.rb

```diff
@@ -0,0 +1,240 @@
+#########################
+###### DEFINITIONS ######
+#########################
+
+
+class SolarSystem
+
+  attr_reader :system_name, :num_planets, :planets, :planet_list
+
+  def initialize(name, planets) 
+    @system_name = name.to_s
+    @planets = {}
+    planets.each {|new_planet| @planets[new_planet.name] = new_planet}
```

Comment:
Style nitpick: in general, you should expand your each loops into multiple lines. So in this case that would be:

```ruby
planets.each do |new_planet|
  @planets[new_planet.name] = new_planet
end
```